src/SFA.DAS.ApprenticeFeedback.Web/Pages/Engagement/Links.cshtml

```diff
@@ -0,0 +1,19 @@
+﻿@page "/links/{linkName}/{feedbackTransactionId}/{apprenticeFeedbackTargetId}"
```

Comment:
@ChrisJWoodcock I've not used razor pages before can you explain what `@page ` is used for ?